packages/cms-cli/commands/server.js

```diff
@@ -1,69 +0,0 @@
-#!/usr/bin/env node
```

Comment:
Removed this file completely.  Are there any concerns over this command?  I think technically it would already be broken if being used since we swapped hs to yargs